src/app/core/components/error/conflict/conflict.component.html

```diff
@@ -1,5 +1,5 @@
-<app-govuk-heading> {{ header || 'This request is no longer available' }}</app-govuk-heading>
-<p id="conflict-body" class="govuk-body" tabindex="0">
+<app-govuk-heading ariaRole="alert"> {{ header || 'This request is no longer available' }}</app-govuk-heading>
```

Comment:
should this be `role="alert"`?

Reply:
No, I'm passing it in to the child heading component as an input. Because 'alert' is a string I don't need to do attribute binding with [ ]. It will be rendered as role="alert" in the DOM if you test it out.

Reply:
I see, then this is all good